From f756930057dc5db84dd6f6309a95cced5e859016 Mon Sep 17 00:00:00 2001
From: zondax <info@zondax.ch>
Date: Mon, 4 Oct 2021 09:59:32 +0000
Subject: [PATCH 1/2] make: do not force qemu selection

---
 framework/proj.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/framework/proj.mk b/framework/proj.mk
index 805d7cc..79fd504 100644
--- a/framework/proj.mk
+++ b/framework/proj.mk
@@ -14,12 +14,6 @@ ifdef QEMU_V7
 #qemu v7 also set?
 $(error Please set EITHER QEMU_V8 OR QEMU_V7 but not both)
 endif
-else
-#qemu v8 not set
-ifndef QEMU_V7
-#qemu v7 also not set
-$(error Plese set either QEMU_V8 or QEMU_V7)
-endif
 endif
 
 ifdef QEMU_V7
-- 
2.17.1

